grafana_client/elements/plugin.py

```diff
@@ -9,60 +11,83 @@ def __init__(self, client):
         self.client = client
         self.logger = logging.getLogger(__name__)
 
-    def get_installed_plugins(self):
+    def list(self):
         """
-        :return:
+        Return list of all installed plugins.
         """
         path = "/plugins?embedded=0"
```

Comment:
Should we allow user to pass the filters like `embedded=0&core=0` ? Can be a separate PR.

Reply:
Sure, why not. Can you point us to corresponding documentation or code, where all the possible filtering parameters are enumerated? Maybe there are even more, about sorting or paging?

Reply:
https://github.com/grafana/grafana/blob/main/pkg/api/plugins.go#L48

Reply:
Thanks, see GH-117.